lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -278,7 +278,9 @@ def add_3d_secure(post, options)
           post[:'3ds_channel'] = '02'
           post[:'3ds_redirect_url'] = three_ds_2_options[:notification_url]
           post[:'3ds_challengewindowsize'] = options[:three_ds_challenge_window_size] || '03'
+          post[:'3ds_version'] = options[:three_ds_version] if options[:three_ds_version]
           post[:d5] = browser_info[:user_agent]
+          post[:'3ds_transtype'] = '01' || options[:transaction_type]
```

Comment:
should we set this as `options[:transaction_type] || '01'` so that if they actually pass in a value it'll use that over '01'

Reply:
yup 🤦‍♀